project_management/turing_binderhub_azure_credit_request_Oct2019.md

```diff
@@ -0,0 +1,67 @@
+# The Turing joining the BinderHub Federation - Credit Request
+
+## Project Title
+
+Turing membership of global MyBinder Federation
+
+## Project PI
+
+Kirstie Whitaker
+
+## Existing Subscription
+
+No - please create.
+Suggested name: turing-mybinder
+
+## Credit Request
+
+$10,000
+
+## Start/End dates for allocation
+
+Begin: 1st October 2019
+End: 30th September 2020
```

Comment:
Is this valid for 1 year only? Why is the allocation starting from 2019? Apologies if you have clarified that somewhere already.

Reply:
Yes it's for one year, aligning with the financial/academic year. It's already been submitted/approved, @KirstieJane just asked that I added it to the repo so it's more findable than the hackmd it was developed in.